compiler/src/iree/compiler/Dialect/HAL/Target/TargetBackend.h

```diff
@@ -149,6 +154,34 @@ class TargetBackend {
     return {};
   }
 
+  // Inserts passes used to configure the `hal.executable.variant` op contents
+  // for translation. The pass manager will be nested on `hal.executable` such
+  // that the pipeline will only run on executable contents.
+  //
+  // The primary purpose of this pipeline is to do the minimum necessary
+  // preprocessing on variants necessary for translation, as well as set any
+  // strategy specific information needed for translation. This is to inform
+  // later passes on how to expand/contract variants for multiple targets that
+  // might be known to translate to the same thing. As such, this pipeline is
```

Comment:
The meaning of "might be known to translate to the same thing" is unclear to me. Could you revise to make it clearer?

Reply:
Let me know if the wording is clearer now.